<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QDialog" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>581</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/fly.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Categories Of Books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horror_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
font-size: 14pt;

</string>
   </property>
   <property name="text">
    <string>Horror</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mystery_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
font-size: 14pt;
</string>
   </property>
   <property name="text">
    <string>Mystery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spiri_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
font-size: 14pt;

</string>
   </property>
   <property name="text">
    <string>Spirituality</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scifi_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
font-size: 14pt;
</string>
   </property>
   <property name="text">
    <string>Sc-Fi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
